Source/Menu/MainForm.cs

```diff
@@ -264,6 +262,31 @@ orderby tool.Text
             }
         }
 
+        private void LoadDocuments(List<ToolStripItem> docs, string folderPath, string code)
```

Comment:
```suggestion
        private void LoadDocuments(List<ToolStripItem> docs, string folderPath, string code = "")
```
Simplify by making the `code` parameter optional.

Reply:
Good idea